clouddriver-web/config/clouddriver.yml

```diff
@@ -255,6 +255,16 @@ google:
 
 openstack:
   enabled: false
+  accounts:
+    - name: test
+      environment: test
+      accountType : v2
+      master: test
```

Comment:
This isn't a test file, but instead a fallback for config properties. Rather than name everything "test" I'd recommend either providing sane defaults, or omitting values you can't generically apply to each client.